src/screens/Customers/AddCustomer.tsx

```diff
@@ -1,38 +1,116 @@
-import React from 'react';
-import { Typography } from '@material-ui/core';
+import { createStyles, FormControl, InputLabel, MenuItem, Select, Theme } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import { RouteComponentProps } from 'react-router-dom';
 import BaseScreen from '../../components/BaseComponents/BaseScreen';
-import TextField from '../../components/TextField';
-import Checkbox from '../../components/Checkbox';
 import Button from '../../components/Button';
-import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
-import { RouteComponentProps } from 'react-router-dom';
+import Checkbox from '../../components/Checkbox';
+import TextField from '../../components/TextField';
+import { CustomerRecord, SiteRecord } from '../../lib/airtable/interface';
+import { createCustomer } from '../../lib/airtable/request';
+import { addCustomer } from '../../lib/redux/siteData';
+import { RootState } from '../../lib/redux/store';
+
+// TODO: Move to utils file?
+const EMPTY_CUSTOMER : CustomerRecord= {
+    name: '',
+    meterNumber: 0,
+    tariffPlansId: '',
+    isactive: false,
+    hasmeter: false,
+    outstandingBalance: '',
+    meterReadingIds: [],
+    meterReadings: [],
+    paymentIds: [],
+    payments: [],
+    customerUpdateIds: [],
+    customerUpdates: [],
+    totalAmountBilledfromInvoices: 0,
+    totalAmountPaidfromPayments: 0,
+}
 
 const styles = (theme: Theme) =>
   createStyles({
     content: {
       color: theme.palette.text.primary,
     },
+    formControl: {
+      width: '100%',
+    },
   });
 
 interface AddCustomerProps extends RouteComponentProps {
-  classes: { content: string };
+  classes: { content: string, formControl: string};
   location: any;
+  currentSite: SiteRecord;
 }
 
+
 function AddCustomer(props: AddCustomerProps) {
-  const { classes } = props;
+  const { classes, currentSite } = props;
+
+  const [selectedTariffPlan, setSelectedTariffPlan] = useState("");
+  const [customerName, setCustomerName] = useState("");
+  const [meter, setMeter] = useState("");
+  const [meterChecked, setMeterChecked] = useState(false);
+  const [customerInactive, setCustomerInactive] = useState(false);
```

Comment:
Not sure how much this really matters, but do you think I should rename these to match the interface/schema? Or does that not matter?

Reply:
Might be very slightly better but as long as the code is readable (which in this case it is) it doesn't matter much. I wouldn't change it.

In fact, changing `customerInactive` to `IsActive` would probably make the code harder to read because then the checkbox would need to be negated, so I like how it is now.